hw01/hw01.py

```diff
@@ -89,4 +94,15 @@ def hailstone(n):
     1
     """
     "*** YOUR CODE HERE ***"
-
+    arr=[]
+    while n>1:
+        arr.append(n)
+        if n%2==0:
+            n=n//2
+        else:
+            n=3*n+1
+    else:
```

Comment:
is this correct?

The functionality is incorrect.  Look at the example and think where should call `print`?

Reply:
have fixed